storybook/pages/UnblockWithPukFlowPage.qml

```diff
@@ -106,11 +106,11 @@ SplitView {
 
         function reset() {
             keycardState = Onboarding.KeycardState.NoPCSCService
-            keycardRemainingPukAttempts = 3
+            keycardRemainingPukAttempts = 5
```

Comment:
we could consider creating some singleton with onboarding constants to keep number like this in single place

Reply:
Yeah, good idea, will do this

Reply:
Done; extracted to `Constants.onboarding`

Reply:
cool, I will move some other constants there as well